libmamba/include/mamba/specs/conda_url.hpp

```diff
@@ -35,7 +35,7 @@ namespace mamba::specs
         explicit CondaURL(util::URL&& url);
         explicit CondaURL(const util::URL& url);
 
-        auto base() const -> const util::URL&;
+        [[nodiscard]] auto generic() const -> const util::URL&;
```

Comment:
I find `base` more meaningful

Reply:
The fact the it the base is not relevant here. What matters is that there is a generic alternative to this class to which we can convert.